booster/library/Booster/Pattern/ApplyEquations.hs

```diff
@@ -16,8 +16,6 @@ module Booster.Pattern.ApplyEquations (
     EquationPreference (..),
     EquationFailure (..),
     EquationTrace (..),
-    pattern CollectEquationTraces,
-    pattern NoCollectEquationTraces,
     eraseStates,
```

Comment:
Am I right that we shouldn't export `EquationTrace` and `eraseStates` any more here?

Reply:
Yes, thanks, removed them